src/lib/eDSL.ml

```diff
@@ -302,18 +301,25 @@ let depends_on l =  Depends_on l
 let on_success_activate n = On_success_activate n
 let on_failure_activate n = On_failure_activate n
 
+type done_when_option = [
+  | `Dependencies_are
+  | `Is_verified of Condition.t
+  | `Product_is 
```

Comment:
Product_is_done and Dependencies_are_done are a little clearer to me (took me a minute to figure out the meaning here).